src/tools/rbd/action/Group.cc

```diff
@@ -721,38 +727,127 @@ int execute_group_snap_list(const po::variables_map &vm,
   if (f) {
     f->open_array_section("group_snaps");
   } else {
+    t.define_column("ID", TextTable::LEFT, TextTable::LEFT);
     t.define_column("NAME", TextTable::LEFT, TextTable::LEFT);
     t.define_column("STATUS", TextTable::LEFT, TextTable::RIGHT);
```

Comment:
```
$ rbd group snap ls data/gp
ID            NAME   STATUS  
104eb2666faf  snap1  complete
```
Since we have changed what is displayed in this column, let's take this opportunity to also change its name to be consistent with the name of the corresponding field:
```suggestion
    t.define_column("STATE", TextTable::LEFT, TextTable::RIGHT);
```

Reply:
Done.  TODO Need to update ReleaseNotes too.